engine/src/main/java/org/terasology/world/block/internal/BlockManagerImpl.java

```diff
@@ -93,7 +93,7 @@ public void initialise(List<String> registeredBlockFamilies,
         if (knownBlockMappings.size() >= MAX_ID) {
             nextId = UNKNOWN_ID;
         } else if (knownBlockMappings.size() > 0) {
-            nextId = (short) knownBlockMappings.size();
+            nextId = knownBlockMappings.values().stream().max(Short::compareTo).orElse((short) 0)+1;
```

Comment:
is there a less costly way to keep track of the max id? If I understand this change correctly, it will iterate through _all_ known block mappings every time a new block/shape is requested?